### ISSUE TITLE
PATCH on ModelResource clears Resource's ImageField data

### ISSUE THREAD

**Issue by hakanb** [django-tastypie/django-tastypie#1018]
I have a model with `description` and `image` fields. The `description` field is defined as a `models.CharField` and the `image` field defined as an `models.ImageField`. If I PATCH the `description` field, the `image` field is emptied. 

This has been discussed on the mailing list but there does not seem to a solution:
https://groups.google.com/forum/#!topic/django-tastypie/cxrI6Cl1z4s

Am I missing something? How do I avoid the `image` field being changed? This seems like an issue that a lot of people would be having.

**Comment by teewuane** [comment]
I agree with sassanh, this can cause a lot of other problems. There are work arounds like mave99a has provided but this seems pretty bad.

**Comment by ketanbhatt** [comment]
Hey, can I submit a PR for this problem? We have been using tastypie for our APIs and didn't notice this bug until we started using S3BotoStorage as our storage backend. 
@SeanHayes let me know if this bug hasn't been fixed for some specific reason and if you would be willing to accept a PR if fixes the problem. Thanks!

**Comment by SeanHayes** [comment]
@ketanbhatt Absolutely! Make sure it has a unit test demonstrating the bug and make sure to follow the other contribution guidelines.
